src/components/RoomSelector.spec.js

```diff
@@ -85,10 +86,12 @@ describe('RoomSelector.vue', () => {
 		const wrapper = shallowMount(RoomSelector)
 
 		expect(axios.get).toHaveBeenCalledWith(
-			generateOcsUrl('/apps/spreed/api/v4/room')
+			generateOcsUrl('/apps/spreed/api/v4/room'),
+			{ params: { includeStatus: true } }
 		)
 
 		// need to wait for re-render, otherwise the list is not rendered yet
+		await flushPromises()
 		await wrapper.vm.$nextTick()
```

Comment:
We probably don't need a $nextTick there anymore, as flushPromises will wait for all the promises to be resolved. Check if that's also working in all tests
```suggestion
```